dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -1059,11 +1068,10 @@ export const Explorer = ({
     isLiveTailOnRef.current,
     patternsData,
     viewLogPatterns,
+    userVizConfigs,
     isPatternConfigPopoverOpen,
     patternRegexInput,
-    userVizConfigs,
```

Comment:
Can this order be changed? This looks like ordered array.

Reply:
fixed it